main/parse.c

```diff
@@ -97,6 +99,19 @@ extern const char *getLanguageName (const langType language)
 	return result;
 }
 
+extern const char getLanguageFileKind (const langType language)
+{
+	char kind;
+
+	Assert (0 <= language  &&  language < (int) LanguageCount);
```

Comment:
... I have no option but some people may think " 0 <= language < LanguageCount  " is easier to read. See another Assert in main/parse.c. All of the take the same style.